### PR TITLE
Add Chrome Apps label to Chrome Apps titles

### PR DESCRIPTION
**Pull request by AmySteam**
Fixes #https://github.com/GoogleChromeLabs/Extension-Docs/issues/136

Changes proposed in this pull request:

- Adds [Chrome apps] to the pages that have an extension equivalent.

### CODE REVIEW

File: site/en/docs/apps/analytics/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: "layouts/doc-post.njk"
-title: "Analytics"
+title: "[Chrome Apps] Analytics"
```
Comment (sebastianbenz) [inline site/en/docs/apps/analytics/index.md:3]: This should go to seoTitle and not title.

Comment (AmySteam) [reply site/en/docs/apps/analytics/index.md:3]: I was going by the search results, which seem to pick the title as opposed to the seoTitle. 

I'll switch to seoTitle for now, merge and see what happens... 🧐

Comment (sebastianbenz) [reply site/en/docs/apps/analytics/index.md:3]: SERPs use the title, but the index might not have been updated.

Comment (AmySteam) [reply site/en/docs/apps/analytics/index.md:3]: Eh... the seoTitle already have Chrome Apps on them... and they still show up in search without the Chrome apps label

Screenshot 2023-07-19 at 13 29 00

image

Hmm...

Comment (sebastianbenz) [reply site/en/docs/apps/analytics/index.md:3]: 🤦 Ian already changed this. The index simply hasn't updated.

Comment (sebastianbenz) [reply site/en/docs/apps/analytics/index.md:3]: Sorry for the confusion./

**Pull request closed without merging**